api.py

```diff
@@ -388,6 +388,107 @@ def set_generic_data(dataset_id, key, data_string, **kwargs):
     return ingest_entities(dataset_id, {'entities': [entity]}, **kwargs)
 
 
+def convert_coordinates(point):
+    if len(point) != 2:
+        raise KeyError('A point should only have two dimensions', point)
```

Comment:
`ValueError` is worth considering too

Reply:
It's a perspective thing right, is it the value of the dict as a whole or the specific key that is in error.

I tried to be judicious about when I used KeyError or ValueError.